src/app/repairwizardpage.h

```diff
@@ -36,10 +36,6 @@ class RepairWizardPage : public AbstractExecutablePage
 public:
     RepairWizardPage(ADTExecutableRunner *run, QWidget *parent = nullptr);
 
-public slots:
-
```

Comment:
No slot containing  enableButtonsAfterChecks() method. Add slot onEnableUiElements containing  enableButtonsAfterChecks() and connect it to appropriate signal.

Reply:
There is no need for the enableButtonsAfterChecks() method, because all the functionality is implemented in the base class AbstractExecutablePage.